src/url.js

```diff
@@ -95,6 +95,18 @@ export function assertHttpsUrl(urlString, elementContext) {
   return urlString;
 }
 
+/**
+ * Asserts that a given url is an absolute HTTP or HTTPS URL.
+ * @param {string} urlString
+ * @return {string}
+ */
+export function assertAbsoluteUrl(urlString) {
+  assert(/^(http\:|https\:)/i.test(urlString),
```

Comment:
A bit of an issue is that `parseUrl` also resolves the relative URL and here I wanted to stress that the URL must be explicitly absolute.